polkadot/node/core/pvf/common/src/worker/security/landlock.rs

```diff
@@ -319,4 +344,57 @@ mod tests {
 
 		assert!(handle.join().is_ok());
 	}
+
+	#[test]
+	fn restricted_thread_cannot_use_tcp_sockets() {
+		// TODO: This would be nice: <https://github.com/rust-lang/rust/issues/68007>.
+		if check_can_fully_enable(LANDLOCK_ABI_FS).is_err() {
+			return
+		}
```

Comment:
Oops, this should be `LANDLOCK_ABI_FS`. I'll wait until after a review to fix it, since this minor change would re-trigger all the CI checks again.